sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Generated/IManagedDatabaseRestoreDetailsOperations.cs

```diff
@@ -0,0 +1,57 @@
+// <auto-generated>
```

Comment:
Can you 
(1) Put a link in your PR description to the PR in the specs repo?
(2) Add the metadata file output from generate.ps1 to `eng\mgmt\mgmtmetadata\<rpName>.txt`

Reply:
Sure. Here is the PR for specs - https://github.com/Azure/azure-rest-api-specs/pull/7172